exemplar/exemplar_integration_test.go

```diff
@@ -0,0 +1,87 @@
+//go:build integration
+
+package exemplar
+
+import (
```

Comment:
Maybe for another PR, or for discussion, but my current understanding is that these modules will be used for smoke testing with a k8s deployed environment to ensure FTL is working well e2e. If that's the case, maybe we should rename this package and have 2 areas for modules like these. Some ideas...

We could rename `exemplar` to `smoketest` and have a smoke test package at the root of the repo. (basically just rename this package)

Then, for other basic examples those could still live in `examples` something like:
```sh
./examples/go/pubsub
./examples/go/cron
./examples/go/http
./examples/kotlin/pubsub
...etc.
```

So the `smoketest` would be a very specific setup that is more hardened and used for testing. Where as `examples` would be a place users could go to see how to do "X" in FTL.

Reply:
I really like this. I'll add that stuff to examples in the next pr.